Matlab/Completion Rules.tmPreferences

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<plist version="1.0">
+<dict>
+    <key>scope</key>
+    <string>source.matlab</string>
+    <key>settings</key>
+    <dict>
+        <key>cancelCompletion</key>
+        <string>^.*\b(?:arguments|break|case|catch|classdef|continue|else|elseif|end|enumeration|events|for|function|global|if|methods|otherwise|parfor|persistent|properties|return|spmd|switch|try|while)$</string>
```

Comment:
I just had a look at #2620 as Python was mentioned in the discussion on discord and found it omitting `^.*` compared to the suggested pattern in #3648. This may (or may not?) have a positive effect on performance, maybe. Thoughts?

```suggestion
        <string>\b(?:arguments|break|case|catch|classdef|continue|else|elseif|end|enumeration|events|for|function|global|if|methods|otherwise|parfor|persistent|properties|return|spmd|switch|try|while)$</string>
```

Reply:
I just tested it, and if I remove the leading `^.*`, the completion cancelling doesn't work anymore for long words, for example when I type `otherwise` (also tested and confirmed with `enumeration`). But it still works for short words, e.g. `else`.
If I keep the `^.*` in the pattern, it seems to work as expected for all words in the pattern.

[matlab-completions.webm](https://user-images.githubusercontent.com/6579999/216699690-23499ba4-7c9b-476a-8734-e6580e543f0d.webm)

Reply:
That's an interesting detail.